CMakeLists.txt

```diff
@@ -127,6 +127,9 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED QUIET)
 list(APPEND DEPLIBS ${CMAKE_THREAD_LIBS_INIT})
 
+# Atomics
+find_package(Atomics REQUIRED)
```

Comment:
since this is linux and riscv/arm specific, this is not globally required. Instead move detection to https://github.com/xbmc/xbmc/blob/master/cmake/scripts/linux/ArchSetup.cmake and add atomics to PLATFORM_REQUIRED_DEPS only when needed